nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -286,4 +325,66 @@ private Type.ClassType typeWithPreservedAnnotations(ParameterizedTypeTree tree)
             type.getEnclosingType(), com.sun.tools.javac.util.List.from(newTypeArgs), type.tsym);
     return finalType;
   }
+
+  public void checkTypeParameterNullnessForAssignabilityForConditionalExpression(
```

Comment:
Eventually, we want the error for conditional expressions to give a different error message than the one for assignments

Reply:
will update it in the end once we are done with the other changes